qutebrowser/commands/command.py

```diff
@@ -33,12 +33,13 @@ class ArgInfo:
 
     """Information about an argument."""
 
-    def __init__(self, win_id=False, count=False, flag=None, hide=False,
-                 metavar=None, completion=None, choices=None):
+    def __init__(self, win_id=False, count=False, hide=False, metavar=None,
+                 zero_count=False, flag=None, completion=None, choices=None):
         if win_id and count:
             raise TypeError("Argument marked as both count/win_id!")
```

Comment:
This is still open - what I meant is that you could accidentally do `@cmdutils.argument('count', zero_count=True)` and forget `count=True` - since `zero_count` makes no sense without `count`, that case should raise an exception and be tested.